pkg/deploy/generator/scripts/gatewayVMSS.sh

```diff
@@ -67,6 +72,19 @@ include /etc/logrotate.d
     create 0600 root utmp
     rotate 1
 }
+
+# Maximum log directory size is 100G with this configuration
+# Setting limit to 100G to allow space for other logging services
+# copytruncate is a critical option used to prevent logs from being shipped twice
+${gateway_logdir} {
+    size 20G
```

Comment:
How did we decide on 100G?

This is more of an lvm question than a suggestion, but I see that https://github.com/Azure/ARO-RP/pull/2978/files#diff-a7a118c3717408a3425f9209b28c891edad1c68d9b9a67cb41b18545f8ce37e3R21 increased disk size, but also decreased the amount of free space given to the root filesystem (+20%FREE rather than +50%FREE), is that understanding correct?

Reply:
I've been looking for input on what size this should be. Is there a maximum time limit between when a line is written to the log, and when that is read by (MDSD?) If so then maybe a larger size is good, but around 20GB?

Correct, the reason that `/` was reduced to 20% is because it is largely unused by the system. IIRC the root partition is around 45GB in size. /var is being increased to 50% of space, the remainder of the disk. Which should be around 200GB in size. Although I wish I had recorded the numbers I took, with all the partitions on disk I don't remember what is left for / and /var.

Reply:
I see, so the change in https://github.com/Azure/ARO-RP/pull/2978 is allocating less to `/`, not `/var`, where the logs are. I was misunderstanding, so with that I think we can get away with 100G.

> Is there a maximum time limit between when a line is written to the log, and when that is read by (MDSD?)

I am not sure on this one.